Program.cs

```diff
@@ -1,10 +1,68 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿// ---------------------------------------------------------------------------------------
+// Training ~ A training program for new joinees at Metamation, Batch- July 2023.
+// Copyright (c) Metamation India.                                                
+// ---------------------------------------------------------------------------------------
+// Program.cs                                                                     
+// Program to print a chess board using unicode values
+// ---------------------------------------------------------------------------------------
+using System.Text;
+using static System.Console;
+
+namespace Training {
+   #region class Program --------------------------------------------------------------------------
+   /// <summary>Chess Board</summary>
+   internal class Program {
+      #region Method ------------------------------------------------
+      /// <summary>This method prints the chess board with all pieces</summary>
+      static void Main () {
+         OutputEncoding = Encoding.UTF8;
+         string vLine = "\u2502", hLine = "\u2500";
+
+         // Black : 265F - pawn, 265C - rook, 265E - knight, 265D - bishop, 265B - queen, 265A - king
+         // White : 2659 - pawn, 2656 - rook, 2658 - knight, 2657 - bishop, 2655 - queen, 2654 - king
+         for (int i = 0; i < 10; i++) {
+            switch (i) {
+               case 0: PrintHrzntLines (hLine, "\u250C", "\u252C", "\u2510"); continue;

```

Comment:
```suggestion
               case 0: PrintHorizontalLines (hLine, "\u250C", "\u252C", "\u2510"); continue;

```
Name properly. Or just PrintHLines is also okay. But this intermediate Hrznt is not good.